src/Symfony/Component/Console/Helper/ProgressBar.php

```diff
@@ -303,17 +303,16 @@ public function iterate(iterable $iterable, int $max = null): iterable
     /**
      * Starts the progress output.
      *
-     * @param int|null $max      Number of steps to complete the bar (0 if indeterminate), null to leave unchanged
-     * @param int      $resumeAt when restarting a previously started progress, set on what step we are restarting so that time estimations are calculated correctly, and the
-     *                           progress is automatically set to the appropriate step
+     * @param int|null $max     Number of steps to complete the bar (0 if indeterminate), null to leave unchanged
+     * @param int      $startAt The starting point of the bar (useful e.g. when resuming a previously started bar)
      */
-    public function start(int $max = null, int $resumeAt = 0): void
+    public function start(int $max = null, int $startAt = 0): void
```

Comment:
Could it break the usage of named parameters? https://www.php.net/manual/en/functions.arguments.php#functions.named-arguments

I wonder if it's applicable here.

Reply:
> .. [10] Parameter names are only covered by the compatibility promise for
constructors of Attribute classes. Using PHP named arguments might break your code when upgrading to newer Symfony versions.

via https://symfony.com/doc/current/contributing/code/bc.html

Reply:
Thanks, my bad I didn't RTFM. :see_no_evil:

Reply:
Also, the original feature was merged a few days ago in a Symfony version (6.2) which hasn't been released yet.